### DISCUSSION TITLE
Copilot Bug - not finishing suggestions with "\n"

### DISCUSSION THREAD

**Discussion by DenverCoder1** [community/community#39277] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Copilot stops suggesting code after an open quote when the suggestion would contain "\n"

![image](https://user-images.githubusercontent.com/20955511/202279424-0ca815b4-41de-4269-a462-1205f906aaa1.png)

Example prompt

```php
<?php

$text = "This is\nan example\nstring.";

// replace all newlines with a \r\n
```

Additionally, there is strange behavior of Copilot suggesting splitting a string to multiple lines rather than using "\n", and no "\n" appearing in the Copilot tab.

![image](https://user-images.githubusercontent.com/20955511/202280925-8f55907d-2898-4c44-8695-ac47314ca9d8.png)

**Comment by cmp-nct** [comment]
I've also had tons of similar bugs when it stops after
code_code_code  . "\r 
It stops instead of  completing the quote

It's especially confusing as it stops in such an awkward place you need more time to think what the code should have been than to rewrite it on your own.
You've to close the quote and re-request the completion.
This reduces the potential quality of the completion and increases the risk of it dreaming something unrelated to the original task.

**Comment by DannyNemer** [comment]
I also encounter this issue; i.e., any Copilot suggestion that contains the sequence `\n` is truncated.

I am surprised this bug has lingered for so long. It seems like an easy fix to escape the text within the VS Code extension.

Here is another example of truncation at `\n`:
<img width="451" alt="image" src="https://user-images.githubusercontent.com/170023/209243558-aac89333-f366-4d20-8f18-a89117e9a295.png">
